typeql-src/modules/ROOT/pages/data/delete.adoc

```diff
@@ -53,13 +53,13 @@ xref:schema/modify.adoc#_undefine_a_type[undefine] to delete types) and can't ha
 - `is` keyword
 ====
 
-== Idempotent deletes
+== Deduplicated deletes
 
-Since TypeDB version 2.22.0 delete queries are idempotent.
+Since TypeDB version `2.22.0`, delete queries are deduplicated.
```

Comment:
```suggestion
Since TypeDB version `2.22.0`, delete queries deduplicate data before performing the delete.
```

Reply:
Fixed as discussed in person with https://github.com/vaticle/typedb-docs/pull/717/commits/f6c252a7ab3c36ac0eb79ebe4ea8ddfe50164d2f